documentation/modules/exploit/solaris/ssh/pam_username_bof.md

```diff
@@ -0,0 +1,102 @@
+## Vulnerable Application
+
+### Description
+
+This module exploits a stack-based buffer overflow in the Solaris PAM
+library's username parsing code, as used by the SunSSH daemon when the
+keyboard-interactive authentication method is specified.
+
+Tested against SunSSH 1.1.5 on Solaris 10u11 1/13 (x86) in VirtualBox,
+VMware Fusion, and VMware Player. Bare metal untested. Your addresses
+may vary.
+
+### Setup
+
+Download and install [Solaris 10u11 1/13
+(x86)](https://download.oracle.com/otn/solaris/10/sol-10-u11-ga-x86-dvd.iso)
```

Comment:
Navigating directly to this link left me with an error from Oracle (after creating an account and logging in) that I hadn't agreed to the OTN License. This is the link that worked for me: https://www.oracle.com/solaris/solaris10/downloads/solaris10-get-jsp-downloads.html (Select Oracle Solaris 10 x86).

Reply:
Oh yeah, I must have already been logged in.